ja_JP/LC_MESSAGES/localeSelection.po

```diff
@@ -1,17 +1,17 @@
 msgid "CHANGE_LANGUAGE"
-msgstr "Change Language"
+msgstr "言語を変更する"
 
 msgid "LANGUAGE_UPDATED_FORMAT"
-msgstr "Your language has been updated to %s."
+msgstr "言語が %sに変更されました。"
 
 msgid "POTENTIAL_CONTRIBUTOR_NOTE"
-msgstr "Sudomemo relies on its community to provide accurate translations. If you see something that does not display correctly, is still in English, is a low-quality translation, or if you would like to add your own language, we invite you to contribute! Full and verified new translations can earn you citizenship. You can do so here: https://github.com/Sudomemo/sudomemo-locales"
+msgstr "Sudomemoは正確な翻訳を提供するためにコミュニティに依存しています。 正しく表示されない、英語で表示される、低品質の翻訳がある、またはあなた自身の言語を追加したい場合は、コミュニティに参加してください！ https://github.com/Sudomemo/sudomemo-localesでこれを翻訳できます。"
 
 msgid "RETURN_TO_CREATORS_ROOM"
 msgstr "Return to Creator's Room"
```

Comment:
"Return to Creator's Room" has not been translated.

This is used in a situation where, after the user has changed their language setting, they are given a button that takes them back to their Creator's Room.